test/runtests.jl

```diff
@@ -373,7 +378,7 @@ end
               (:document, :distance, :indices, :query, :query_name, :corpus_uuid,
                :document_uuid)
         @test sprint(explain, (first(Tables.rowtable(tbl)))) ===
-              "The query \" other\" exactly matched the text \"There were other cobras \".\n"
+              "The query \" other\" exactly matched the text \" There were other cobras \".\n"
```

Comment:
Does the fact that this change broke an existing test mean that it's technically a breaking change?

Reply:
Hm. Maybe? We can do a minor bump. Since this is a `sprint` test it’s basically printing, but the fact is someone could be relying on not having that initial space in their `Document`s somehow.

Reply:
Hm yeah, perhaps a minor bump would be better then but I'll defer to your judgement on what's best.

Reply:
Bumped!